Dockerfile

```
@@ -6,15 +6,16 @@ FROM node:22-alpine AS base
 
 WORKDIR /usr/src/app
 
-ENV YARN_DISABLE_GIT_HOOKS=1
 ENV CI=true
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
 
 RUN apk add --no-cache dumb-init g++ make python3
 
-COPY --chown=node:node yarn.lock .
+RUN corepack enable
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)